### ISSUE TITLE
Feature request: Missing show/hide feature for form components

### ISSUE THREAD

**Issue by anujshivapatel** [Adobe-Marketing-Cloud/aem-core-wcm-components#19]
It is common requirement to be able to show/hide a form field on the basis of values of other fields (e.g. showing a textbox to input email id if send email checkbox is selected). This feature is present in WCM form foundation components (/libs/foundation/components/form). However, it is missing from core components version (/apps/core/wcm/components/form). 
It would be a really nice feature for core form components, as there could be more customers looking to upgrade from old form foundation components (non Touch-UI based).

**Comment by hkuiyo** [comment]
Indeed, +1, this is very frequently requested and it would be a pain to create custom code for it all the time

**Comment by gabrielwalt** [comment]
To provide a good show/hide features for the from components is on our roadmap, as well as removing the dependency on the foundation components.

**Comment by andreeadracea** [comment]
This feature request is related to the issue:
https://github.com/adobe/aem-core-wcm-components/issues/417
